preprocess.sh

```diff
@@ -70,4 +70,6 @@ mkdir ${DATA}/bin
 # Since we only provide training data, you may want to make your own split.
 fairseq-preprocess --source-lang $src --target-lang $tgt \
     --trainpref $DATA/train-${outf} --validpref $DATA/dev-${outf} --testpref $DATA/test-${outf} \
-    --destdir ${DATA}/bin/ --workers 20 --dataset-impl raw
+    --destdir ${DATA}/bin/ --workers 20 --dataset-impl raw \
+    # --srcdict checkpoints/${src}_${tgt}/dict.${src}.txt \
```

Comment:
Extra comments

Reply:
I actually want this comment here, as it could be useful.